lib/natural/sentiment/SentimentAnalyzer.js

```diff
@@ -42,42 +42,45 @@ const englishNegations = require("./English/negations_en.json").words;
 const spanishNegations = require("./Spanish/negations_es.json").words;
 const dutchNegations = require("./Dutch/negations_du.json").words;
 
-
 // Mapping from type of vocabulary to language to vocabulary
 var languageFiles = {
-  "afinn" : {
-    "English": [englishAfinnVoca, englishNegations],
-    "Spanish": [spanishAfinnVoca, spanishNegations]
+  afinn: {
+    English: [englishAfinnVoca, englishNegations],
+    Spanish: [spanishAfinnVoca, spanishNegations]
   },
-  "senticon": {
-    "Spanish": [spanishSenticonVoca, spanishNegations],
-    "English": [englishSenticonVoca, englishNegations],
-    "Galician": [galicianSenticonVoca, null],
-    "Catalan": [catalanSenticonVoca, null],
-    "Basque": [basqueSenticonVoca, null]
+  senticon: {
+    Spanish: [spanishSenticonVoca, spanishNegations],
+    English: [englishSenticonVoca, englishNegations],
+    Galician: [galicianSenticonVoca, null],
+    Catalan: [catalanSenticonVoca, null],
+    Basque: [basqueSenticonVoca, null]
   },
-  "pattern": {
-    "Dutch": [dutchPatternVoca, dutchNegations],
-    "Italian": [italianPatternVoca, null],
-    "English": [englishPatternVoca, englishNegations],
-    "French": [frenchPatternVoca, null]
+  pattern: {
+    Dutch: [dutchPatternVoca, dutchNegations],
+    Italian: [italianPatternVoca, null],
+    English: [englishPatternVoca, englishNegations],
+    French: [frenchPatternVoca, null]
   }
 };
 
-
 class SentimentAnalyzer {
-
   constructor(language, stemmer, type) {
     this.language = language;
     this.stemmer = stemmer;
 
-    this.vocabulary = languageFiles[type][language][0];
+    // this.vocabulary must be a copy of the languageFiles object
+    // or in subsequent execution the polarity will be undefined
+    // shallow copy - requires ES6
+    this.vocabulary = Object.assign(
+      {},
+      require(languageFiles[type][language][0])
```

Comment:
using require statement like this as an expression breaks the use in web browser (for instance with WebPack). I will adapt this to <code>languageFiles[type][language][0]</code>